### PR TITLE
Plans: cancel the google-vouchers test.

### PR DESCRIPTION
**Pull request by retrofox**
Fix: #6865 

This PR removes all mention of ad vouchers from the plans and plans/compare pages, both from the sidebar and from the NUX. It also removes it from the redesigned plans page. Google vouchers should now be launched for all premium and business sites from the my-plan page however.

EDIT: Important, I should mention I'm taking this opportunity to remove the wordpress-ad-credits feature as well. It was a subset of the google-vouchers feature... basically brings the $100 advertising voucher to $200 for business. But we've never launched it, and given the results of the google-voucher, I think we need to rethink it. So just ripping it out for now.

### Current Design
image2

image3

### Redesign

image

### Current Design
image2

image3

### Redesign
image

### Testing

Make sure the changes look and work as expected. These changes affect the following pages:

`plan-features` flag **disabled**
* `http://calypso.localhost:3000/plans/<your-testing-site>`
* `http://calypso.localhost:3000/plans/compare/<your-testing-site>`
* `http://calypso.localhost:3000/start/plans`
* `http://calypso.localhost:3000/start/plans/compare`

Run this normally `make run` and also with the redesign flag enabled `ENABLE_FEATURES=manage/plan-features make run`

Also make sure the google voucher feature is still available on the my-site page for a premium and business site.

Test live: https://calypso.live/?branch=update/google-vouchers-on-the-plan

### CODE REVIEW

File: config/development.json [hunk 1]
```diff
@@ -102,7 +101,6 @@
 		"perfmon": false,
 		"persist-redux": true,
 		"plans/personal-plan": true,
-		"plans/wordpress-ad-credits": true,
```
Comment (mtias) [inline config/development.json]: Is this related?

Comment (rralian) [inline config/development.json]: Sorry, it's related because it's another part of the "advertising vouchers" feature that never launched. I'm removing it in this PR as well, as it uses much of the same plumbing.

File: client/lib/plans/constants.js [hunk 2]
```diff
@@ -107,17 +107,14 @@ export const plansList = {
 		getStoreSlug: () => PLAN_PREMIUM,
 		availableFor: ( plan ) => includes( [ PLAN_FREE, PLAN_PERSONAL ], plan ),
 		getDescription: () => i18n.translate( 'Your own domain name, powerful' +
-			' customization options, easy monetization with WordAds and lots of space for audio and video.' ),
-		getDescriptionWithGoogleVouchers: () => i18n.translate( 'Your own domain name, powerful' +
-			' customization options, easy monetization with WordAds, lots of space for audio and video, and $100 advertising voucher.' ),
+			' customization options, easy monetization with WordAds, and lots of space for audio and video.' ),
```
Comment (retrofox) [inline client/lib/plans/constants.js]: the parentheses `)` should be in its own line

**Comment by apeatling** [comment]
Going through the UI, I am not seeing anything out of place.

**Comment by retrofox** [comment]
Tested all routes in both modes and LGTM.

**Comment by retrofox** [comment]
I think we should rebase this branch before to merge.
